torchgeo/datasets/etci2021.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ETCI 2021 dataset."""
+
+import glob
+import os
+import shutil
+from typing import Callable, Dict, List, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class ETCI2021(VisionDataset):
+    """ETCI 2021 Flood Detection dataset.
+
+    The `ETCI2021 <https://nasa-impact.github.io/etci2021/>`_
+    dataset is a dataset for flood detection
+
+    Dataset features:
+    * 33,405 VV & VH Sentinel-1 Synthetic Aperture Radar (SAR) images
+    * 2 binary masks per image representing water body & flood, respectively
+    * 2 polarization band images (VV, VH) of 3 RGB channels per band
+    * 3 RGB channels per band generated by the Hybrid Pluggable
+    Processing Pipeline 'hyp3'
```

Comment:
I think this needs to be indented a couple spaces, otherwise it will end the bulleted list, same below

Reply:
Thanks for catching this. I was unsure how this would show up in the docs when rendered.

Reply:
Fixed